```
(Link target) -> 
  kernel-development.lib(mode_development.obj) : error LNK2019: unresolved external symbol "unsigned int MCstartupstack_length" (?MCstartupstack_length@@3IA) referenced in function "public: enum IO_stat __thiscall MCDispatch::startup(void)" (?startup@MCDispatch@@QAE?AW4IO_stat@@XZ) [C:\vulcan\Community\worker\try-community-x86-win32\worktree\build-win-x86\livecode\engine\test-kernel-development.vcxproj]
  kernel-development.lib(mode_development.obj) : error LNK2019: unresolved external symbol "unsigned char * MCstartupstack" (?MCstartupstack@@3PAEA) referenced in function "public: enum IO_stat __thiscall MCDispatch::startup(void)" (?startup@MCDispatch@@QAE?AW4IO_stat@@XZ) [C:\vulcan\Community\worker\try-community-x86-win32\worktree\build-win-x86\livecode\engine\test-kernel-development.vcxproj]
  C:\vulcan\Community\worker\try-community-x86-win32\worktree\build-win-x86\livecode\Fast\test-kernel-development.exe : fatal error LNK1120: 2 unresolved externals [C:\vulcan\Community\worker\try-community-x86-win32\worktree\build-win-x86\livecode\engine\test-kernel-development.vcxproj]
```

--- a/engine/src/dummystartupstack.cpp
+++ b/engine/src/dummystartupstack.cpp
@@ -1,8 +1,5 @@
-extern "C"
-{
 
-char *MCstartupstack;
-unsigned int MCstartupstack_length;
+unsigned char MCstartupstack[] = { '\0' };
+unsigned int MCstartupstack_length = 0;
 
-}
 

